packages/react-native/src/Authenticator/common/DefaultContent/types.ts

```diff
@@ -61,7 +62,7 @@ export type DefaultContentProps<
   'error' | 'Footer' | 'isPending'
 > & {
   buttons: DefaultButtons;
-  body?: React.ReactNode;
+  body?: React.ReactNode | string;
   fields: FieldsType[];
   headerText?: string;
```

Comment:
Can we go ahead and make this required now that we've added `headerText` to all the subcomponents?

Reply:
Done!